### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by R0Wi**

### CODE REVIEW

**Review by bahnwaerter — CHANGES_REQUESTED** [review]
Please take a look at my comments and try to resolve them.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: R0Wi
```
[the remark this answers is not in the document]
Comment (R0Wi) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: This was generated by Github so I expect it to work :-)

**Review by bahnwaerter — APPROVED** [review]
Now, the template looks good to me. Feel free to merge this change.

**Comment by R0Wi** [comment]
Thx :-)

**Pull request merged**